src/yanfeishao/cs555/constant/ErrorInfo.java

```diff
@@ -51,7 +51,7 @@
     /**
      * The constant US21_FAMILY.
      */
-    public static final String US21_FAMILY = "Husband in family should be male and wife in family should be female";
+    public static final String US21_FAMILY = "Husband in family [%s] should be male [%s] and wife in family should be female [%s]";
     /**
```

Comment:
constant US21_FAMILY is no longer used it is replaced with the constant 
US21_PERSON which has the requested information.

On 10/18/2015 10:44 PM, JackieDreamy wrote:
> In src/yanfeishao/cs555/constant/ErrorInfo.java [1]:
> 
>> @@ -51,7 +51,7 @@
>> /**
>> * The constant US21_FAMILY.
>> */
>> - public static final String US21_FAMILY = "Husband in family should
> be male and wife in family should be female";
>> + public static final String US21_FAMILY = "Husband in family [%s]
> should be male [%s] and wife in family should be female [%s]";
>> /**
> 
> Same above
> th the
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/JackieDreamy/AgileSprint/pull/49#discussion_r42332960
> [2] https://github.com/JackieDreamy/AgileSprint/pull/49/files#r42332960